.gitignore

```diff
@@ -15,6 +15,7 @@
 /tmp/*
 !/log/.keep
 !/tmp/.keep
+/coverage
```

Comment:
nitpick that this isn't relevant to the commit

Reply:
Fair enough, I've moved it into a separate commit.